loky/backend/spawn.py

```diff
@@ -329,7 +329,7 @@ def freeze_support():
     It should be called right after the beginning of the programme, to
     avoid recursive process spawning.
     """
-    if len(sys.argv) >= 2 and sys.argv[1] == "--multiprocessing-fork":
+    if len(sys.argv) >= 2 and sys.argv[1] == "--loky-fork":
```

Comment:
Shouldn't we rename this to loky-spawn BTW? Or do we want to keep something closer to the multiprocessing terminology?

Reply:
I don't have a strong opinion on this one so as you want.

Reply:
Let's keep the focus on the original problem :)